scripts/zones/West_Sarutabaruta_[S]/mobs/Ramponneau.lua

```diff
@@ -0,0 +1,68 @@
+-----------------------------------
+-- Area: West Sarutabaruta (S)
+--   NM: Ramponneau
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_ADD_EFFECT,mob:getShortID());
+    mob:setMobMod(MOBMOD_AUTO_SPIKES,mob:getShortID());
+    mob:addStatusEffect(EFFECT_SHOCK_SPIKES,10,0,0);
+    mob:getStatusEffect(EFFECT_SHOCK_SPIKES):setFlag(32);
+end;
+
+-----------------------------------
+-- onMobFight Action
+-----------------------------------
+
+function onMobFight(mob,target)
+    mob:SetMobAbilityEnabled(false); -- not currently working
+    mob:setTP(0);  -- temp until above is fixed
+end;
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+
+function onAdditionalEffect(mob,target,damage)
+    
+    local power = math.random(4,15);
+    local params = {};
+    params.bonusmab = 0;
+    params.includemab = false;
+    power = addBonusesAbility(mob, ELE_ICE, target, power, params);
+    power = power * applyResistanceAddEffect(mob,target,ELE_ICE,0);
```

Comment:
Shucks, sorry @waytim i forgot to tell you mob additional effect isn't the same stuff as item additional effects.

And I don't see the scripting for spikes here, you used the mobmod to enable scripted spikes. I'll PM you examples for both in IRC.